### ISSUE TITLE
Model not training on large dataset

### ISSUE THREAD

**Issue by lalitr994** [microsoft/unilm#486]
Hi,
I have trained Layoutlm v2 funsd model with around 5K images with 30K steps. after successful training inference is not working correctly(not even a single correct prediction).
but the same setup is working if I trained around 2.5 images. I am not sure what I am missing.

**Comment by sathwikacharya** [comment]
Hey
I am having a similar issue where the logits outputted by the model after training is nan. Any reason why this is happening?
I am training this on a custom dataset with 29 classes and 40000 data points. The steps followed are identical to this link except for a few tweaks : https://github.com/NielsRogge/Transformers-Tutorials/blob/master/LayoutLMv2/RVL-CDIP/Fine_tuning_LayoutLMv2ForSequenceClassification_on_RVL_CDIP.ipynb

I do not know if this is a problem but I am training it on multiple gpus using the Accelerate API of huggingface. Any help to this is much appreciated

**Comment by nissansz** [comment]
hi, where to get or pretrain the models for Japanese, Korean, etc.? steve8000818@gmail.com
